src/types/UserLoginProps.ts

```diff
@@ -0,0 +1,10 @@
+interface UserLoginProps {
```

Comment:
Also, the name could also be generalized here, some preparation for a more general application state would benefit of having some UserProperties instead of a very specific UserLoginProps